net/lftp/Makefile

```diff
@@ -8,13 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lftp
-PKG_VERSION:=4.7.5
+PKG_VERSION:=4.8.2
 PKG_RELEASE:=1
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=http://lftp.tech/ftp/ \
 		ftp://ftp.st.ryukoku.ac.jp/pub/network/ftp/lftp/
-PKG_HASH:=53b5c5eba2e38b418bcf451776f2df596050dff4db90ab4ea73d872f8b1fd0d8
 
+PKG_MD5SUM:=36f3ca2c070391282e0260dd6234af3b
```

Comment:
Drop MD5SUM, it's deprecated

Reply:
It is for compatibility with older Openwrt, does it support also PKG_HASH now ?